lib/organization/postfeed.dart

```diff
@@ -1,203 +1,302 @@
-                              width: 80.w,
-                            ),
-                            Icon(Icons.comment),
-                            SizedBox(
-                              width: 80.w,
-                            ),
-                            Icon(Icons.share),
-                          ]),
-                        ),
-                        SizedBox(
-                          height: 5.h,
-                        ),
-                        Text(
-                          "lorem ipsum dolor sit amet, consectetur adipis elit.",
-                          style: TextStyle(fontWeight: FontWeight.w700, fontSize: 14.sp),
-                        ),
-                        SizedBox(
-                          height: 10.h,
-                        ),
-                      ],
-                    ),
-                  ]),
-                ),
-              )),
-            ));
-  }
-}
+import 'package:cura/shared/widgets/navigation-bar.dart';
+import 'package:favorite_button/favorite_button.dart';
+import 'package:flutter/material.dart';
+import 'package:cura/individual/home_page_individual.dart';
+import 'package:flutter_screenutil/flutter_screenutil.dart';
+import 'package:share/share.dart';
+import 'package:social_share/social_share.dart';
+
+class postfeed extends StatefulWidget {
+  static const String routeName = '/postfeed';
+  const postfeed({Key? key}) : super(key: key);
+
+  @override
+  State<postfeed> createState() => _postfeedState();
+}
+
+class _postfeedState extends State<postfeed> {
+  @override
+  Widget build(BuildContext context) {
+    return ScreenUtilInit(
+        designSize: const Size(428, 926),
+        builder: (context, child) => Scaffold(
+              bottomNavigationBar: const CustomNavigationBar(currentIndex: 1),
+              body: SafeArea(
+                  child: SizedBox(
+                width: double.infinity,
+                child: Container(
+                  decoration: const BoxDecoration(
+                      gradient: LinearGradient(
+                    colors: [
+                      Color(0xFF6CAFB4),
+                      Colors.white,
+                      Colors.white,
+                      Color(0xFF6CAFB4),
+                    ],
+                    begin: Alignment.topCenter,
+                    end: Alignment.bottomCenter,
+                    tileMode: TileMode.clamp,
+                  )),
+                  child: SingleChildScrollView(
+                    child: Column(children: [
+                      Padding(
+                        padding: EdgeInsets.all(10.sp),
+                        child: Row(children: [
+                          IconButton(
+                            icon: Icon(
+                              Icons.arrow_back_ios_new,
+                              size: 15.w,
+                            ),
+                            onPressed: () {
+                              Navigator.of(context)

```

Comment:
Hmmm...:thinking: I think pushing the HomePage here, would lead to building up in the stack, which will degrade the performance of the app. 
Don't you think we should **pop** the page when back_arrow icon is clicked?